hostplumber/api/v1/hostnetwork_types.go

```diff
@@ -33,10 +33,16 @@ type HostNetworkSpec struct {
 type HostNetworkStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	OvsStatus       []*OvsStatus       `json:"ovsStatus,omitempty"`
 	InterfaceStatus []*InterfaceStatus `json:"interfaceStatus,omitempty"`
 	Sysctl          []string           `json:"sysctlConfig,omitempty"`
 }
 
+type OvsStatus struct {
+	NodeInterface string `json:"nodeInterface,omitempty"`
```

Comment:
Should we have this as a list instead of a string? As bridge can accommodate multiple ports/interfaces.

Reply:
Talked to Arjun about this and we want to keep just one interface per bridge. If using bonding, the bond config should be created separately and then the bond interface can be listed here.